catalog.html

```diff
@@ -0,0 +1,298 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Седона</title>
+    <link rel="stylesheet" href="styles/style.css">
+  </head>
+  <body>
+    <header>
+      <span class="visually-hidden home">
+      <a class="navigation-logo" href="index.html">
+        На главную
+      </a>
+    </span>
+      <nav class="navigation">
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a class="navigation-link" href="index.html">Главная</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">О седоне</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="catalog.html">Гостиницы</a>
+          </li>
+        </ul>
+      </nav>
+      <ul class="navigation-user">
+        <li class="navigation-item">
+          <a class="navigation-link" href="#">
+              <span class="visually-hidden">
+                 Поиск
+              </span>
+          </a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="#">
+              <span class="visually-hidden">
+                Избранное
+           </span></a>
+        </li>
+        <li class="navigation-item">
+          <button name="want">Хочу сюда</button>
+        </li>
+      </ul>
+    </header>
+
+    <main class="main-catalog">
+      <section class="catalog-header">
+        <section>
+          <h1 class="catalog-header-title  ">Гостинницы Седоны</h1>
```

Comment:
```suggestion
          <h1 class="catalog-header-title  ">Гостиницы Седоны</h1>
```
Чтобы не допускать ошибок и опечаток, текст лучше копировать непосредственно из макета:
![image](https://user-images.githubusercontent.com/17790771/170198125-19caae07-33ed-40df-a47d-c0ea4c76ba90.png)